homework3/Exercises/1-kalkulaator_ver2/Program.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace _1_kalkulaator_ver2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("See on kalkulaator, millega saab liita, lahutada, korrutada, jagada ja astendada omavahel arve. Väljumiseks sisestage 'exit'");
+            Console.WriteLine("Komakoha jaoks kasutage punkti '.'\n");
+            do 
+            {
+                try 
+                {
+                    Beginning: //kui programmis on goto Beginning;, siis programm tuleb siia, et algusest peale alustada.
```

Comment:
Forget that GOTO statements exists! There is almost no reason ever-ever why you should use GOTO's (it's considered extremely bad practice)